corehq/apps/hqcase/tests/test_dbaccessors.py

```diff
@@ -180,6 +180,7 @@ class ESAccessorsTest(TestCase):
 
     def setUp(self):
         super(ESAccessorsTest, self).setUp()
+        register_alias(CASE_INDEX_INFO.index, CASE_INDEX_INFO)
```

Comment:
The `register_alias()` (and friends) are more verbose because there are tests that require "non-standard registries", and I was trying to keep this "registry change set" as small as possible (i.e. attempting to avoid overhauling every ES test out there).  I think another iteration of refactoring could make more improvements, which will happen. Right now there are ongoing communications about whether or not we still need this registry at all.